Bad archive base_deps_windows_native_static_09.tar.gz

The archive at https://tmp.kiwix.org/ci/base_deps_windows_native_static_09.tar.gz seems to be corrupted.

```
$ tar tzf base_deps_windows_native_static_09.tar.gz 
SOURCE/icu4c-73.2/.extract_ok
SOURCE/pugixml-1.2/.extract_ok
...
... /* omitting some output */
....
BUILD_win-amd64/INSTALL/bin/icuuc74.dll
BUILD_win-amd64/INSTALL/bin/testplug.dll
BUILD_win-amd64/libcurl/.install_ok

gzip: stdin: decompression OK, trailing garbage ignored
BUILD_win-amd64/xapian-core-1.4.26/.test_ok
BUILD_win-amd64/zlib-1.2.12/.compile_ok
BUILD_win-amd64/docoptcpp-0.6.2/.install_ok
ARCHIVE/.download_boostregex_ok
ARCHIVE/.download_libcurl_ok
ARCHIVE/.download_docoptcpp-0.6.2_ok
tar: Child returned status 2
tar: Error is not recoverable: exiting now
```

There is no such problem with its [predecessor](https://tmp.kiwix.org/ci/base_deps_windows_native_static_08.tar.gz).

And it causes CI to fail. For example - https://github.com/kiwix/kiwix-build/actions/runs/11545702517/job/32132991525

```
Run python .github\\scripts\\ensure_base_deps.py
--- Getting archive base_deps_windows_native_static_09.tar.gz ---
D:\a\kiwix-build\kiwix-build\.github\scripts\ensure_base_deps.py:55: DeprecationWarning: Python 3.14 will, by default, filter extracted tar archives and reject files or modify their metadata. Use the filter argument to control this behavior.
  f.extractall(str(HOME))
Traceback (most recent call last):
  File "D:\a\kiwix-build\kiwix-build\.github\scripts\ensure_base_deps.py", line 72, in <module>
    main()
  File "D:\a\kiwix-build\kiwix-build\.github\scripts\ensure_base_deps.py", line 55, in main
    f.extractall(str(HOME))
  File "C:\hostedtoolcache\windows\Python\3.12.7\x64\Lib\tarfile.py", line 2302, in extractall
    self._extract_one(tarinfo, path, set_attrs=not tarinfo.isdir(),
  File "C:\hostedtoolcache\windows\Python\3.12.7\x64\Lib\tarfile.py", line 2365, in _extract_one
    self._extract_member(tarinfo, os.path.join(path, tarinfo.name),
  File "C:\hostedtoolcache\windows\Python\3.12.7\x64\Lib\tarfile.py", line 2448, in _extract_member
    self.makefile(tarinfo, targetpath)
  File "C:\hostedtoolcache\windows\Python\3.12.7\x64\Lib\tarfile.py", line 2502, in makefile
    copyfileobj(source, target, tarinfo.size, ReadError, bufsize)
  File "C:\hostedtoolcache\windows\Python\3.12.7\x64\Lib\tarfile.py", line 251, in copyfileobj
    buf = src.read(bufsize)
          ^^^^^^^^^^^^^^^^^
  File "C:\hostedtoolcache\windows\Python\3.12.7\x64\Lib\gzip.py", line 324, in read
    return self._buffer.read(size)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\hostedtoolcache\windows\Python\3.12.7\x64\Lib\_compression.py", line 68, in readinto
    data = self.read(len(byte_view))
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\hostedtoolcache\windows\Python\3.12.7\x64\Lib\gzip.py", line 537, in read
    uncompress = self._decompressor.decompress(b"", size)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
zlib.error: Error -3 while decompressing data: invalid stored block lengths
............
Error: Process completed with exit code 1.```

--- a/.github/workflows/cd.yml
+++ b/.github/workflows/cd.yml
@@ -5,7 +5,7 @@ on:
    tags:
      - r_[0-9]+
  schedule:
-   - cron: '0 1 * * *'
+   - cron: '0 3 * * *'
 
 jobs:
   Windows:

